firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteMutationQueue.java

```diff
@@ -288,13 +307,17 @@ public MutationBatch getNextMutationBatchAfterBatchId(int batchId) {
                 + "AND dm.batch_id = m.batch_id "
                 + "ORDER BY dm.batch_id");
 
+    int rowsRead[] = new int[] {0};
+
     List<MutationBatch> result = new ArrayList<>();
     Set<Integer> uniqueBatchIds = new HashSet<>();
     while (longQuery.hasMoreSubqueries()) {
       longQuery
           .performNextSubquery()
           .forEach(
               row -> {
+                ++rowsRead[0];
```

Comment:
Would it make sense to just make this a part of forEach and similar methods (and the return value of the method)?

Just spitballing. Feel free to ignore.